greptimedb/results/_query_results/_m6i.8xlarge_bluesky_1m.query_results

```diff
@@ -0,0 +1,72 @@
+------------------------------------------------------------------------------------------------------------------------
```

Comment:
It is okay to include query results only for the 1m dataset. Query result files for the 10m, 100m and 1000m data sets can be deleted.